release notes/v0.57.0.md

```diff
@@ -0,0 +1,172 @@
+k6 `v0.57.0` is here 🎉! This release includes:
+
+- Introducing helpers for functional testing.
+- The `k6 new` command now supports templates and ProjectIDs.
+- The `k6/experimental/csv` module gets a new `asObjects` option.
+- We no longer support the `k6/experimental/browser` module.
```

Comment:
```suggestion
- We no longer support the `k6/experimental/browser` module, in favor of `k6/browser`.
```

I think reading this without mentioning the graduated module feels a little bit weird, like we're dropping browser support. What do you think of this suggestion? 🤔